### ISSUE TITLE
A panic occurred in the storage during the query

### ISSUE THREAD

**Issue by orangle** [VictoriaMetrics/VictoriaMetrics#4887]
### Describe the bug

panic: runtime error: invalid memory address or nil pointer dereference

### To Reproduce

tag query

### Version

v1.93.0  https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.93.0-cluster
 

### Logs

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x77888e]

goroutine 358425298 [running]:
gitlab.corp.qunar.com/cm/VictoriaMetrics/lib/storage.(*Storage).getDeletedMetricIDs(...)
        .../VictoriaMetrics/lib/storage/storage.go:315
gitlab.corp.qunar.com/cm/VictoriaMetrics/lib/storage.(*indexSearch).searchTagValueSuffixesForPrefix(0xc132caaa00, 0xc130568e88, 0x20?, {0xc1ec703310, 0x48, 0x50}, 0x36, 0x40?, 0x1)
        .../VictoriaMetrics/lib/storage/index_db.go:1231 +0x6e
gitlab.corp.qunar.com/cm/VictoriaMetrics/lib/storage.(*indexSearch).searchTagValueSuffixesForDate(0xc132caaa00, 0x0?, 0x4c89, {0x0, 0x0}, {0xc2458d4100, 0x36}, 0x2?, 0x100000000000e?)
        .../VictoriaMetrics/lib/storage/index_db.go:1224 +0x52d
gitlab.corp.qunar.com/cm/VictoriaMetrics/lib/storage.(*indexSearch).searchTagValueSuffixesForTimeRange.func1(0xc23b8e28c0?)
        .../VictoriaMetrics/lib/storage/index_db.go:1179 +0x196
created by gitlab.corp.qunar.com/cm/VictoriaMetrics/lib/storage.(*indexSearch).searchTagValueSuffixesForTimeRange
        .../VictoriaMetrics/lib/storage/index_db.go:1175 +0x16f
```

### Used command-line flags

ARGS="-storageDataPath=data \
-bigMergeConcurrency=6 \
-denyQueriesOutsideRetention=true \
-http.disableResponseCompression=true \
-rpc.disableCompression=true \
-search.maxTagKeys=10000 \
-search.maxTagValueSuffixesPerSearch=10000 \
-loggerTimezone=Asia/Shanghai"

**Comment by zekker6** [comment]
Hello @orangle 
This issue was discovered yesterday - https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4873. It has been fixed in [v1.93.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.93.1).
Please, note that this issue could lead to data loss so it is recommended to restore data from backup in case any data will be missing.

**Comment by hagen1778** [comment]
@orangle could you please show logs preceding the panic?

**Comment by orangle** [comment]
> Hello @orangle This issue was discovered yesterday - #4873. It has been fixed in [v1.93.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.93.1). Please, note that this issue could lead to data loss so it is recommended to restore data from backup in case any data will be missing.

We will first upgrade to see if it can resolve the issue. Thank you.

**Comment by hagen1778** [comment]
Closing as stale. Feel free to reopen if issue still exists
